### PR TITLE
refactor: Update public facing form output

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

- Updates FIP to include bottom blue border + stretch to the width of the page
- Updates content pages to stretch out the text 
- Adds date modified to the form output.  

- Removes route utils + unused classes.

| Before                                          | After                                        |
| ----------------------------------------------- | -------------------------------------------- |
| current | form-output |

# Test instructions | Instructions pour tester la modification

Check form output

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

Are there any related issues or tangent features you consider out of scope for
this issue that could be addressed in the future? If so please create issues and provide
links in this section

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [ ] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [ ] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: next-env.d.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```
Comment (thiessenp-cds) [inline next-env.d.ts:3]: Was this change intentional? I'm not sure if it matters either way but I remember a note (I think) about not modifying this file

Comment (timarney) [reply next-env.d.ts:3]: I'll revert that.  It always ends up in my builds.

Comment (thiessenp-cds) [reply next-env.d.ts]: Same.. no idea why and it's kind of annoying

**Review by thiessenp-cds — APPROVED** [review]
Nice, work - LGTM :)

File: next-env.d.ts (hunk 1 above)
Comment (timarney) [reply next-env.d.ts]: @dsamojlenko @bryan-robitaille  --- I reverted the file ... I've been pulling it out of PRs for quite a while now.

Do ya'll end up with that file edited as well?  Thinking we should just commit it if you do.
